packages/truffle-interface-adapter/lib/ethereum-overloads.ts

```diff
@@ -1,11 +1,21 @@
 import BN from "bn.js";
-import Web3 from "web3";
+import { Web3Shim } from "./web3-shim";
+
+export const EthereumDefinition = {
+  async initNetworkType (web3: Web3Shim) {
+    // truffle has started expecting gas used/limit to be
+    // hex strings to support bignumbers for other ledgers
+    getBlock(web3);
```

Comment:
Naming note here, not to change for this PR, but we might want to fix at some point:

For explicitness purposes, these methods should probably include the word `override` or something similar, to convey that it's not that we're getting the block, but we're overriding how we get the block.

An alternative solution would be to abstract this, so that `getBlock`, `getTransaction`, would be in a mapping, something like:
```typescript
const overrides = {
  "getBlock": (original) => {
    const _oldFormatter = original.method.outputFormatter;
    original.method.outputFormatter = (block) => {
      _oldFormatter.call(original, block);
      /* ... */
    };
  },
  /* ... */
}
```

Just if you want to keep this in the back of your mind for whenever.